### ISSUE TITLE
Invalid API Key Error on production on 1.1.3 or 1.1.4

### ISSUE THREAD

**Issue by KeisukeNagakawa** [giorgiabosello/google-maps-react-markers#20]
First of all, thank you for this great library!

**Describe the bug**
I'm runnning with this package `^1.1.2` on nextjs ver 13 and when running it locally, it's okay. but when running it on procuction(uploaded to vercel server), invaid api key  error occurred.

When runnning with ver `1.1.2` it fixed.
The error on production started on two or three days ago so I noticed that is because of this upgrade.

**To Reproduce**
Run it with `^1.1.2` and deploy to vercel => error
With `1.1.2`, okay.

**Expected behavior**
Map rendered cooreectly.

**Codesandbox**
Sorry, no.

**Comment by giorgiabosello** [comment]
@KeisukeNagakawa I can't help without any code. You mention 1.1.3 and 1.1.4 on the title, but 1.1.2 on the issue.
Are you sure your env with the API is read on production? 
It doesn't seem a problem of the library.

**Comment by KeisukeNagakawa** [comment]
@giorgiabosello Thanks for your reply, and sorry for my mistype about the title. I think this error is related 1.1.4 or 1.1.3. 1.1.4 has higher possiblity of being this error because the time it started to cause error and upgrade was almost the same.

The code is like this:
```
        <GoogleMap
          apiKey={process.env.NEXT_PUBLIC_GOOGLE_MAP_API_KEY}
          defaultCenter={defaultCenter}
          defaultZoom={12}
          mapMinHeight="95vh"
          onGoogleApiLoaded={onGoogleApiLoaded}
          options={options}
          callback="initMap"

          // libraries={["places", "marker"]}
        >
          {spots.map((spot) => {
            return (
              <Marker
                key={spot.id}
                markerId={spot.id}
                lat={spot.geopoint.latitude}
                lng={spot.geopoint.longitude}
              />
            );
          })}
        </GoogleMap>

```

it caused invalid api key error. I hard-coded this env var, stopped all restriction about the google map api key ( referrer, api library, ...), and checked that the api key is linked to my billing account which is also linked to my valid creditcard. Also, I changed the key to another project's key but this error occurred. 

Considering to these circumstances, it is highly likely that your latest commit has some problem related only to the "production" environment on vercel (though it is hard to come up with these kind of error)


Hope this helps!

**Comment by giorgiabosello** [comment]
@KeisukeNagakawa there's difference between 1.1.3 and 1.1.4. 
1.1.3 had a bug in the script string, fixed in 1.1.4 (issue #19)

Another thing: the `callback` prop you are passing is not valid.

**Comment by KeisukeNagakawa** [comment]
callback was my mistake(I editted to remove it)

**Comment by KeisukeNagakawa** [comment]
@giorgiabosello I checked it and found that the problem exists in ver 1.1.3's [this code](https://github.com/giorgiabosello/google-maps-react-markers/compare/1.1.2...1.1.3#diff-06cb1b2f53486fb3c361ea9ceedb61c7f1f5f6321b1f76f99bf0495db6f5a9bdR11) which is lately fixed in ver 1.1.4. 

The reason why it was ok for localhost and not okay on production is unclear, but anyway, I confirmed that ver 1.1.4 works good. It's also good that it surpresses error message related to callback.

So it is now all okay, thanks!
